lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -1798,8 +1798,9 @@ describe('GatherRunner', function() {
         .mockResponse('Page.getInstallabilityErrors', {installabilityErrors: []});
 
       const result = await GatherRunner.getWebAppManifest(passContext);
-      expect(result).toHaveProperty('raw', JSON.stringify(manifest));
-      expect(result && result.value).toMatchObject({
+      const parsedManifest = result ? result.parsedManifest : null;
+      expect(parsedManifest).toHaveProperty('raw', JSON.stringify(manifest));
```

Comment:
this can be reverted to after you revert the changes in `gather-runner`, but I think an assertion here for `result.url` would be good.